src/components/shared/select/SelectComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import "./Select.scss";
+import Select from "@mui/material/Select";
+import MenuItem from "@mui/material/MenuItem";
+import InputLabel from "@mui/material/InputLabel";
+import { useState } from "react";
+
+export interface SelectProps {
+  selectName: string;
+  data: string[];
+}
+
+const styles = {
+  minWidth: 250,
+};
+
+export const SelectComponent = ({ selectName, data }: SelectProps) => {
+  const [selectValue, setSelectValue] = useState("");
+
+  const handleSelect = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSelectValue(e.target.value as string);
+  };
+
+  return (
+    <>
+      <InputLabel>{selectName}</InputLabel>
+      <Select
+        sx={styles}
+        label="None"
```

Comment:
why is it hardcoded?

Reply:
changed. but I used "none" as it was default value before user select any other option.